### ISSUE TITLE
Content Review from Design System Accessibility Working Group

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#60096]
## Design System Accessibility Working Group
A coalition of accessibility specialists working with the design system to establish an accessibility strategy ahead of working with USWDS.

- Team name: 
-	OCTO-DE product owner: @humancompanion-usds , @artsymartha68 
- Accessibility specialist: (Full list in doc)
 
### What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [x] Something else 
 
### Give a brief description of what your team needs support with:

My team needs support with reviewing an [accessibility design system strategy document](https://docs.google.com/document/d/11-ZLC69Ijk1Ai7FVxzFhIQ6TmcWU-QEtqRiCkS7uuXI/edit?usp=sharing) with the intent on publishing on design.va.gov.

[Given the content team has done extensive work with their content style guide](https://design.va.gov/content-style-guide/), we hope to get early feedback on how to develop the maturity of accessibility guidelines within the design system as well.
 
 
## Timeframe
This is not a Veteran facing product

### When does this work need to be completed?
- Estimated launch date: ASAP, in hopes of publishing a working document
- Content, Accessibility, and IA work needed by: June 23, 2023 (ideally)
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

[VA.gov design system accessibility strategy v.0 ](https://docs.google.com/document/d/11-ZLC69Ijk1Ai7FVxzFhIQ6TmcWU-QEtqRiCkS7uuXI/edit?usp=sharing)

#### Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [x] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

 **Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.
 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by sara-amanda** [comment]
## CAIA Inquiry
- **This appears to have been addressed (or partially addressed) via this ticket:** Publish Accessibility strategy to design.va.gov #598 (closed by Matt D. on July 12, 2023)
- Ref. Jill Taylor's notes in that ticket: 
> - (7/6/2023) I spoke to Matt after meeting with Josh Kim. I will update the Accessibility standards and maturity model section of the document. Once approved by Matt, we will find a home for the content on design.va.gov and publish.
>  - (7/11/2023) Sent final draft for review/feedback to Martha Wilkes via Slack. Once approved, I'll write a ticket for Matt to publish content in the "About" section on design.va.gov.
>  -  Accessibility standards and maturity model section of the document.
- Ref. [a11y-specialists thread 6/21/2023](https://dsva.slack.com/archives/C0335Q88X0S/p1687376202352169) 
### Next Steps
@joshkimux are you needing additional work from CAIA on this ticket? I see that @SarahKay8 worked on this with you, [along with other a11y specialists](https://dsva.slack.com/archives/C0335Q88X0S/p1687439366105669?thread_ts=1687377848.782899&cid=C0335Q88X0S). Just us know if additional support is needed, or if you would like to close out this ticket.

cc: @coforma-terry

**Comment by sara-amanda** [comment]
@coforma-terry moving to "landed" in projects and "validate" in ZenHub to sync the two up.

**Comment by joshkimux** [comment]
We can close this as it has been published 🙏
